Makefile

```diff
@@ -0,0 +1,48 @@
+XDG_CACHE_HOME = $(HOME)/.cache
+XDG_CONFIG_HOME = $(HOME)/.config
+XDG_DATA_HOME = $(HOME)/.local/share
+
+all: install-brew \
+	install-sh \
+	install-bash \
+	install-git \
+	install-emacs \
+	install-nvm \
+	install-vim \
+	install-vint
+
+install-bash:
+	@echo "Installing bash..."
+	stow -v -R -t ~ bash
```

Comment:
Might be worth checking these args and if always wanting them then make a function that always calls them

Reply:
Ignoring for now. Will check later.

Reply:
Closing as created issue.